split.Console/Models/Blackjack/BlackjackCard.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+namespace split.Console.Models.Blackjack
+{
+    public class BlackjackCard
```

Comment:
Agreed. There are a couple classes that would benefit from having an interface between games, this being one of them